community/cloud-foundation/templates/iam_member/iam_member.py

```diff
@@ -17,12 +17,13 @@
 def generate_config(context):
     """ Entry point for the deployment resources. """
 
-    project_id = context.env['project']
+    project_id = context.properties.get('projectID', context.env['project'])
```

Comment:
Shouldn't this be `projectId`? This seems to be common among the other templates.
```suggestion
    project_id = context.properties.get('projectId', context.env['project'])
```